src/interfaces/stepper-interfaces/CreateMemorandum/create-memorandum-resource-interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface CreateMemorandumResourceIF {
+  helpSection: {
+    header: string
+    helpText: any
```

Comment:
Is this an object? or maybe an array of strings?

Reply:
`helpText` is an object